elastic/shared/runners/datastream.py

```diff
@@ -50,6 +50,33 @@ async def create(es, params):
     return ops, "ops"
 
 
+class DeleteRemoteDataStream(Runner):
+    multi_cluster = True
+
+    def __init__(self):
+        super().__init__()
+
+    async def __call__(self, multi_es, params):
+        """
+        Deletes specified datastreams from all clusters
+        """
+        ops = 0
+        for cluster_name, cluster_client in multi_es.items():
```

Comment:
given we always invoke `delete_data_stream` (and we raise on failure), isn't `ops` always `len(multi_es.items())`? Alternatively, if you wanted a slicker way to do this, you could omit the increment `ops +=1` below and instead do: 
```suggestion
        for ops, (cluster_name, cluster_client) in enumerate(multi_es.items()):
```

Reply:
Nice suggestion! Addressed in https://github.com/elastic/rally-tracks/pull/333/commits/73c2bf289d88ddb3fa35c7f092dd7cffdbf59fc8